digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -593,7 +610,20 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService) extends
      }
   }
 
-  get("/:assetType") {
+  //Description of Api entry point to get all assets by asset type and municipality
+  val getAssetsByTypeMunicipality =
+    (apiOperation[Long]("getAssetsByTypeMunicipality")
+      .parameters(
+        queryParam[Int]("municipality").description("Municipality Code when we will execute the search by specific asset type"),
```

Comment:
"Municipality Code when we will execute the search by specific asset type" instead of using `when` you could use `where`.